lib/hider/accounts/triagram.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Hider.Accounts.Trigram do
```

Comment:
This is not a proper implementation of trigrams (I think), but it's optimized for understanding, and it works.